tools/ui-components/src/toggle-button/types.ts

```diff
@@ -0,0 +1,13 @@
+export type ButtonVariant = 'primary' | 'danger';
+
+export type ButtonSize = 'small' | 'medium' | 'large';
+
+export interface ToggleButtonProps {
+  children: React.ReactNode;
+  size?: ButtonSize;
+  variant?: ButtonVariant;
+  disabled?: boolean;
+  checked?: boolean;
+  onChange?: (value: boolean) => void;
+  className?: string;
+}
```

Comment:
CC, @ahmadabdolsaheb, this is a story of the `toggleButton as TB` component, right?

Because there is two toggleButton, which confused me

https://github.com/rajgaur98/freeCodeCamp/blob/929c923823011ce5af4b8988be1646125f034e93/client/src/components/helpers/toggle-button.tsx#L11-L74?plain=1

Reply:
Yes, that the bootstrap one. Once we have replaced that, we will unify all of the toggle buttons and pull them from the same component.